charts/victoria-metrics-agent/CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## Next release
 
-- TODO
+**Update note: VictoriaMetrics components image tag template have been changed, tag suffix like `-scratch`, `-cluster`, `-enterprise` can be specified in `.Values.<component>.image.variant`, `.Values.<component>.image.tag` can be omitted if it's the same as `.Chart.AppVersion`.
```

Comment:
Rephrased a bit to avoid making an impression that this is a breaking change:
```suggestion
**Update note**: The VictoriaMetrics components image tag template has been updated. This change introduces the option to specify tag suffixes like `-scratch`, `-cluster`, `-enterprise` using `.Values.<component>.image.variant`. Additionally, you can now omit `.Values.<component>.image.tag` to automatically use the version specified in `.Chart.AppVersion`.